Examples/pixmaps.lisp

```diff
@@ -0,0 +1,172 @@
+;;;;  Copyright (c) 2020 Daniel Kochmański
+;;;;
+;;;;    License: BSD-2-Clause.
+
+(defpackage #:clim-demo.pixmaps
+  (:use #:clim-lisp #:clim)
+  (:export #:pixmaps))
+(in-package #:clim-demo.pixmaps)
+
+;;; This demo tests copying data between different mediums. Select a region on
+;;; a pane by dragging a pressed mouse pointer and paste it by clicking on a
+;;; target pane. That will lead to the invocation of MEDIUM-COPY-AREA. Copied
+;;; data is not persistent - it is a low level protocol that copies pixels
+;;; between drawables, so when the sheet is repainted (manually or by the WM),
+;;; that data may be erased. The third pane is an exception, because it is
+;;; repainted from a pixmap.
+;;;
+;;; The "Source" pane has displayed example data, the "Target" pane is another
+;;; ordinary pane while the "Pixmap" pane represents the allocated pixmap -
+;;; the pixmap itself is not visible because it is an off-screen window.
+;;; Copying and pasting on the "Pixmap" pane in fact copies from a pixmap and
+;;; pastes to a pixmap, and that pixmap data is copied to the pane with
+;;; MEDIUM-COPY-AREA.
+;;;
+;;; As a bonus, this demo illustrates how to use blank-area for drag-and-drop
+;;; transformations with a McCLIM-specific extension allowing to specify the
+;;; sheet and to obtain both sheet and coordinates of the blank-area object.
```

Comment:
Can you put some of this into a label pane in the demo frame, like the "file manager" demo does?

Reply:
fixed